.github/workflows/testdockertask.yml

```diff
@@ -0,0 +1,39 @@
+# Test docker-related tasks on linux & windows
+
+name: Test Dockertask
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true # until we fix tests on windows
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest] # docker not available on macOS
+        python-version: [3.7, 3.8]
+        install: [min-req, install, develop, sdist, wheel, style]
+      fail-fast: false
+
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }} on ${{ matrix.os }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Update build tools
+      run: python -m pip install --upgrade pip==20.2.1 setuptools==30.2.1 wheel
```

Comment:
I wonder if we should keep this hardcoded here. We probably should test (at least for one env) other pip/setuptools (as we did in travis). @satra - any opinion?